paddle/operators/transpose_op.cc

```diff
@@ -72,17 +72,29 @@ Transpose Operator.
 The input tensor will be permuted according to the axis values given.
 The op functions is similar to how numpy.transpose works in python.
 
-For example: input = numpy.arange(6).reshape((2,3))
-the input is:
-array([[0, 1, 2],
-      [3, 4, 5]])
-given axis is: [1, 0]
-
-output = input.transpose(axis)
-then the output is:
-array([[0, 3],
-       [1, 4],
-       [2, 5]])
+For example:
+
+    .. code-block:: python
```

Comment:
`python` or `text`，please try to compare the preview

Reply:
Thx. `text` is more fitting here.